File: agri-analysis-server/app/routes/predict.py
import json

from flask import Blueprint, jsonify, request

from app.spark import df_reader
import pyspark.sql.functions as f

bp = Blueprint('predict_bp', __name__)


@bp.route('/predict', methods=['POST'])
def get_predict_data():
    data = json.loads(request.get_data())
    print(data)
    return '2323'

File: agri-analysis-client/src/api/predict.js
import request from '@/utils/request';

export async function getPredictData(dateRange, marketId, varietyId) {
  const { data } = await request({
    url: '/api/predict',
    data: {
      dateRange,
      marketId,
      varietyId
    },
    method: 'post'
  });

  return data;
}
